config/management/genesis/src/storage_helper.rs

```diff
@@ -57,10 +57,7 @@ impl StorageHelper {
             .import_private_key(APTOS_ROOT_KEY, Ed25519PrivateKey::generate(&mut rng))
             .unwrap();
         storage
-            .import_private_key(CONSENSUS_KEY, Ed25519PrivateKey::generate(&mut rng))
-            .unwrap();
-        storage
-            .import_private_key(EXECUTION_KEY, Ed25519PrivateKey::generate(&mut rng))
+            .set(CONSENSUS_KEY, Ed25519PrivateKey::generate(&mut rng))
```

Comment:
This CONSENSUS_KEY is Ed25519, but the one in `impl TestValidator` in `aptos-move/vm-genesis/src/lib.rs` was a BLS key... I find this confusing...

Reply:
good catch, I think this is not used anymore so no test covers it